requirements-app.txt

```diff
@@ -7,5 +7,5 @@ itsdangerous==0.24 # pyup: ignore
 git+https://github.com/alphagov/digitalmarketplace-utils.git@46.3.1#egg=digitalmarketplace-utils==46.3.1
 
 # Elasticsearch 5.0
-elasticsearch==5.5.3 # pyup: >=5.0.0,<6.0.0 # recommended by https://github.com/elastic/elasticsearch-py/blob/master/README
+elasticsearch==6.3.1 # pyup: >=5.0.0,<6.0.0 # recommended by https://github.com/elastic/elasticsearch-py/blob/master/README
```

Comment:
For backwards compatibility, we can install both versions of the client.

```
elasticsearch5==5.5.3
elasticsearch==6.3.1
```

Reply:
If we only care about backwards compatibility for migrating, we don't actually need two versions of the client; the functional tests pass with any combination of client and server versions.

Reply:
Even better.